books/RayTracingInOneWeekend.html

```diff
@@ -483,23 +483,15 @@
 <div class='together'>
 Let’s add a single object to our ray tracer. People often use spheres in ray tracers because
 calculating whether a ray hits a sphere is pretty straightforward. Recall that the equation for a
-sphere centered at the origin of radius $R$ is $x^2 + y^2 + z^2 = R^2$. The way you can read that
-equation is “for any $(x, y, z)$, if $x^2 + y^2 + z^2 = R^2$ then $(x,y,z)$ is on the sphere, and
-otherwise it is not”. It gets uglier if the sphere center is at
-$(\mathbf{c}_x, \mathbf{c}_y, \mathbf{c}_z)$:
+sphere centered at the origin is $x^2 + y^2 + z^2 = R^2$.
+It gets uglier if the center is at $\mathbf{C}$:
 
-  $$ (x-\mathbf{c}_x)^2 + (y-\mathbf{c}_y)^2 + (z-\mathbf{c}_z)^2 = R^2 $$
+  $ (x-\mathbf{c}_x)^2 + (y-\mathbf{c}_y)^2 + (z-\mathbf{c}_z)^2 = R^2 $
 </div>
 
 <div class='together'>
 In graphics, you almost always want your formulas to be in terms of vectors so all the x/y/z stuff
-is under the hood in the `vec3` class. You might note that the vector from center
-$\mathbf{c} = (\mathbf{c}_x,\mathbf{c}_y,\mathbf{c}_z)$ to point $\mathbf{P} = (x,y,z)$ is
-$(\mathbf{p} - \mathbf{c})$, and therefore
-
-  $$ (\mathbf{p} - \mathbf{c}) \cdot (\mathbf{p} - \mathbf{c})
-     = (x-\mathbf{c}_x)^2 + (y-\mathbf{c}_y)^2 + (z-\mathbf{c}_z)^2
-  $$
+is under the hood in the `vec3` class.
```

Comment:
If this derivation is removed, the equation at line 500 is left as an exercise for the reader. I don't see a reason to pull this explanation.